result.nim

```diff
@@ -5,6 +5,11 @@
 # at your option. This file may not be copied, modified, or distributed except according to those terms.
 
 type
+  # TODO ValueError is a Defect - is that what we want here really? It's what
```

Comment:
I've hit this before too, should not be a defect.
However, in Nim v.0.20 this is changed to CatchableError:
https://github.com/nim-lang/Nim/commit/87232a38e4fc40a3acb154a78bf32f8ea5734d13

Reply:
Yeah, there was a bunch of such fixes done by GULPH (quite a lot of Exception types are wrong in 0.19.6).